src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -24,7 +25,7 @@ public static HtmlEncoder Create(params UnicodeRange[] allowedRanges)
         }
     }
 
-    public sealed class DefaultHtmlEncoder : HtmlEncoder
+    internal sealed class DefaultHtmlEncoder : HtmlEncoder
```

Comment:
+1 there is no harm keeping those signatures public and that it takes ages to switch internal->public. For example: VS2013' JSLS.dll. All methods in language service are internal, therefore WebEssentials cannot extend for JS supersets preview panes and provide matching features that are available for CSS-superset Less/Sass editors: https://github.com/madskristensen/WebEssentials2013/issues/374. Same the case with VS2015's JSLS. This is definitely a rare use-case, but when it is required and the assembly has method marked as internal, the most unpleasant experience.